Sources/NIOHTTP2/HTTP2ChannelHandler+InlineStreamMultiplexer.swift

```diff
@@ -201,3 +205,50 @@ extension NIOHTTP2Handler {
         }
     }
 }
+
+extension InlineStreamMultiplexer {
+    func setStreamChannels(_ streamChannels: any ContinuationSanitizer) {
+        self.commonStreamMultiplexer.setStreamChannels(streamChannels)
+    }
+}
+
+extension NIOHTTP2Handler {
+    /// A variant of `NIOHTTP2Handler.StreamMultiplexer` which creates a child channel for each HTTP/2 stream and yields it to the supplied async continuation.
+
+    /// In general in NIO applications it is helpful to consider each HTTP/2 stream as an
```

Comment:
I assume these are meant to be joined:

```suggestion
    /// A variant of `NIOHTTP2Handler.StreamMultiplexer` which creates a child channel for each HTTP/2 stream and yields it to the supplied async continuation.
    ///
    /// In general in NIO applications it is helpful to consider each HTTP/2 stream as an
```